src/iptax/workday/scraping.py

```
@@ -37,22 +43,41 @@ async def navigate_to_time_page(
+
+    # Detect layout: sidebar (Personal button) or hamburger (MENU button).
+    # Now that page is confirmed loaded, a short timeout is safe.
     personal_button = driver.get_by_role("button", name="Personal", exact=True)
-    await personal_button.wait_for(state="visible", timeout=ELEMENT_TIMEOUT)
-    await personal_button.hover()
-    logger.info("Hovered over Personal button, looking for Time link...")
-
-    # Find and click the "Time" link in the revealed submenu
+    try:
+        await personal_button.wait_for(state="visible", timeout=_NAV_LAYOUT_TIMEOUT)
+    except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)